R/create_eq.R

```diff
@@ -25,35 +26,40 @@ create_eq.default <- function(lhs, rhs, ital_vars, use_coefs, coef_digits,
     rhs$final_terms <- add_greek(rhs, rhs$final_terms, greek, intercept, raw_tex)
   }
 
-  # Add error row
+  # Add error row or not in lm
+  if (!use_coefs){
   error_row <- rhs[nrow(rhs) + 1,]
   error_row$term <- "error"
   error_row$final_terms <- "\\epsilon"
-  rhs <- rbind(rhs, error_row)
+  rhs <- rbind(rhs, error_row)}
```

Comment:
Super minor but can you bump this close curly brace to the next line?

Reply:
Fixed.